schema/v0.7/schema.json

```diff
@@ -0,0 +1,238 @@
+                        "$id": "#root/nodeList/items/longitude", 
+                        "title": "Longitude", 
+                        "type": "number",
+                        "examples": [
+                            -79.49186460024643
+                        ],
+                        "default": 0.0
+                    },
+                    "mode": {
+                        "$id": "#root/nodeList/items/mode", 
+                        "title": "Mode", 
+                        "type": "string",
+                        "default": "",
+                        "examples": [
+                            "ap",
+                            "station",
+                            "adhoc",
+                            "802.11s"
+                        ],
+                        "pattern": "^.*$"
+                    },
+                    "model": {
+                        "$id": "#root/nodeList/items/model", 
+                        "title": "Hardware Model", 
+                        "type": "string",
+                        "default": "",
+                        "examples": [
+                            "er-x-sfp",
+                            "lap-120"
+                        ],
+                        "pattern": "^.*$"
+                    },
+                    "name": {
+                        "$id": "#root/nodeList/items/name", 
+                        "title": "Name", 
+                        "type": "string",
+                        "default": "",
+                        "examples": [
+                            "sn1a1"
+                        ],
+                        "pattern": "^.*$"
+                    },
+                    "router": {
+                        "$id": "#root/nodeList/items/router", 
+                        "title": "Router", 
+                        "type": "string",
+                        "default": "",
+                        "examples": [
+                            "sn1r1"
+                        ],
+                        "pattern": "^.*$"
+                    },
+                    "ssid": {
+                        "$id": "#root/nodeList/items/ssid", 
+                        "title": "SSID", 
+                        "type": "string",
+                        "default": "",
+                        "examples": [
+                            "tomesh-airmaxac-sn1a1"
+                        ],
+                        "pattern": "^.*-.*-.*$"
+                    },
+                    "status": {
+                        "$id": "#root/nodeList/items/status", 
+                        "title": "Status", 
+                        "type": "string",
+                        "default": "active",
+                        "examples": [
+                            "active",
+                            "inactive", 
+                            "proposed"
+                        ],
+                        "pattern": "^active|inactive|proposed$"
```

Comment:
A lot of the values are strictly validated now. This allows UI designer to handle only a subset of the infinite possibilities called `string`.